hypervisor/include/dm/vioapic.h

```diff
@@ -120,5 +143,6 @@ struct acrn_single_vioapic *vgsi_to_vioapic_and_vpin(const struct acrn_vm *vm, u
 /**
  * @}
  */
+
```

Comment:
Pls help remove the out-of-date documentation of `vioapic_set_irqline_lock` and `vioapic_set_irqline_nolock` in `vioapic.h`. Thanks.